setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup
+
+
+with open("README.md", "r") as desc:
+    long_description = desc.read()
+
+
+setup(
+	name="xtractmime",
+	version="1.0.0",
+	description="Package for sniffing MIME(Multipurpose Internet Mail Extension) types",
+	long_description=long_description,
+	long_description_content_type="text/markdown",
+	author="Akshay Sharma",
+	author_email="akshaysharmajs@gmail.com",
+	url="https://github.com/scrapy/xtractmime.git",
+	packages=["xtractmime"],
+	python_requires="",
+	classifiers=[
+					
+				]
```

Comment:
It does not hurt to fill these out.

For `python_requires`, given there’s `--ignore-requires-python`, I think we can use 3.6, since 3.5 reached end-of-life already.